docker/scripts/requirements.txt

```diff
@@ -1,5 +1,3 @@
-Paver==1.3.4
-Sphinx==1.8.1
```

Comment:
What happens to our Sphinx build if we remove this dependency?

Reply:
It should still work, since `pip install -e .` will install Sphinx because it is in `requirements.txt`